work/manas/index.html

```diff
@@ -1,21 +1,522 @@
----
-student: Name
-layout: blank.liquid
-tags:
-- students
----
-
+<!--
```

Comment:
Please do not comment this out. This is how the framework knows to display your page link on our home page.